notebooks/tutorials/landscape_evolution/erosion_deposition/erosion_deposition_component.ipynb

```diff
@@ -0,0 +1,266 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Parameters\n",
+    "nrows = 41\n",
+    "ncols = 41\n",
+    "dx = 100.0\n",
+    "K = 0.0001 # erodibility coefficient, 1/yr\n",
+    "m_sp = 0.5 # exponent on drainage area or discharge, -\n",
+    "n_sp = 1.0 # exponent on slope, -\n",
+    "sp_crit = 0.0 # erosion threshold\n",
+    "v_s = 100.0 # settling velocity parameter (dimensionless if drainage area is used instead of discharge)\n",
+    "F_f = 0.5 # fraction of fines generated during bed erosion\n",
+    "initial_elevation = 200.0 # starting elevation of an \"uplifted block\" (rapid baselevel drop), m\n",
+    "\n",
+    "run_duration = 120000.0 # duration of run, yr\n",
+    "dt = 100.0 # time-step duration, yr\n",
+    "plot_every = 40000.0 # time interval for plotting, yr\n",
+    "\n",
+    "# Derived parameters\n",
+    "nsteps = int(run_duration / dt)\n",
+    "next_plot = plot_every\n",
+    "\n",
+    "# set up colormap\n",
+    "cmap = copy.copy(mpl.cm.get_cmap(\"pink\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create a grid with one side open:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mg = HexModelGrid((nrows, ncols), spacing=dx, node_layout='rect', orientation='vertical')\n",
+    "z = mg.add_zeros('topographic__elevation', at='node')\n",
+    "# add some roughness, as this lets \"natural\" channel planforms arise\n",
+    "np.random.seed(0)\n",
+    "initial_roughness = np.random.rand(z.size)\n",
+    "z[:] += initial_roughness\n",
+    "z[mg.core_nodes] += initial_elevation\n",
+    "z[mg.boundary_nodes] = 0.0\n",
+    "\n",
+    "# close off boundaries on 3 sides\n",
+    "is_closed_boundary = np.logical_and(mg.status_at_node != mg.BC_NODE_IS_CORE,\n",
+    "                                    mg.x_of_node < (np.amax(mg.x_of_node) - 0.5 * dx)\n",
+    "                                   )\n",
+    "mg.status_at_node[is_closed_boundary] = mg.BC_NODE_IS_CLOSED"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Instantiate components:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fr = FlowAccumulator(mg, depression_finder='DepressionFinderAndRouter')\n",
+    "ed = ErosionDeposition(mg,\n",
+    "                       K=K,\n",
+    "                       m_sp=m_sp,\n",
+    "                       n_sp=n_sp,\n",
+    "                       sp_crit=sp_crit,\n",
+    "                       v_s=v_s,\n",
+    "                       F_f=F_f,\n",
+    "                       solver='adaptive'  # use the adaptive time stepper, which is slightly faster\n",
+    "                      )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the model in a loop with topographic uplift:"
```

Comment:
This is a little confusing. Is incremental uplift added with each pass through the loop? If so, I don't see where.

If instead what you mean is that you're going to eroded an uplifted block (that is no longer actively uplifting), you could change the wording to "Run the model in a loop to evolve the uplifted block."

Reply:
oops! fixed